client/scripts/view.js

```diff
@@ -59,3 +59,221 @@ const displayCheckout = (body, total, checkout) => {
+        submitButton.addEventListener('click', () => {
+            let prodID = button.getAttribute('product_number');
+            editEvent(prodID)
+
+        });
+    })
+
+    const submitEvent = function () {
+        formData = new Object();
+        formData.id = Math.floor(Math.random() * 1000000);
+        formData.title = document.getElementById("product-title").value;
+        formData.description = document.getElementById("product-description").value;
+        formData.price = document.getElementById("product-price").value;
+        formData.image = document.getElementById("product-image").value;
+        if (formData.image == "") {
+            formData.image = "./assets/default_img.png";
+        }
+        //console.log(formData);
+        postNewProduct(formData);
+        window.localStorage.setItem("formData", JSON.stringify(formData))
+    }
+
+    const editEvent = function (prodID) {
+        editData = new Object();
+        editData.id = prodID;
+        editData.title = document.getElementById("editProduct-title").value;
+        editData.description = document.getElementById("editProduct-description").value;
+        editData.price = document.getElementById("editProduct-price").value;
+        editData.image = document.getElementById("editProduct-image").value;
+        //console.log(editData);
+        postEditProduct(editData);
+    }
+
+    document.getElementById("saveOrder").addEventListener("click", function () {
+        let data = window.localStorage.getItem('prodIdlayout');
+        postLayout(data);
+    });
+
+    [...document.querySelectorAll('.removeProdBut')].forEach(function (item) {
+        item.addEventListener('click', function () {
+            console.log("REMOVE PRODUCT!");
+        });
+    });
+
+    [...document.querySelectorAll('.editProduct')].forEach(function (item) {
+        item.addEventListener('click', async function () {
+            let prodID = item.getAttribute('product_number')
+            console.log(prodID + " - EDIT PRODUCT! from edit button");
+            let product = await getProductById(prodID)
+            console.log("Product info from server for editting " + product.id);
+            
+            // NEED TO GET PRODUCT'S INFORMATION
+            document.getElementById("editProduct-title").value = product.title;
+            document.getElementById("editProduct-description").value = product.description;
+            document.getElementById("editProduct-price").value = product.price;
+        });
+    });
+
+    document.querySelector('.grid').addEventListener('click', function (e) {
+        if (elementMatches(e.target, '.card-remove, .card-remove i')) {
+            postRemove(e);
+            console.log("sent remove post");
+        }
+    });
+
+}
+
+window.onload = (event) => {
+    loadAdminPage();
+    //startPage();
+    //document.getElementById("start").addEventListener("click", function () { loadAdminPage()});
+};
```

Comment:
Admin should have its own model, like Store, and the html page would reference the admin model for onload.